src/molecules/formfields/ToggleField/Readme.md

```diff
@@ -2,7 +2,14 @@
   <ToggleField
     htmlFor=''
     label='Toggle Choices'
-    toggleChoices={[ 'Choice 1', 'Choice 2' ]}
+    toggleChoices={[
+      {label: 'Choice 1', value: 1},
+      {label: 'Choice 2', value: 2}
+    ]}
     tooltipMessage='You can choose either of these awesome options!'
+    input={{
+      value: state.value,
+      onChange: (value) => setState({value})
```

Comment:
I didn't know you could do this with `setState`. Do you have a link to docs/an article about it. I can't find anything currently